mlir_graphblas/src/test/GraphBLAS/lower_transpose.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: graphblas-opt %s | graphblas-opt --graphblas-lower | FileCheck %s
+// RUN: graphblas-opt %s | graphblas-opt --graphblas-structuralize --graphblas-lower | FileCheck %s
```

Comment:
Ideally, we would only test the `--graphblas-structuralize` piece, verifying that it adds in the correct `convert_layout` calls. I suppose we could also test that `graphblas-lower` ignores bad input. But going forward, we should remove all tests which do a line-by-line comparison of lowering beyond graphblas (i.e. down to SCF). All those tests should be replaced with functional tests. That doesn't need to happen with this PR -- just a note for the future.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/d5e602fea2f3e2514222e7b65c8dbda36922fbd6 adds a transpose functional test and lowering test. It also uses `tersify_mlir` for readability. See https://github.com/metagraph-dev/mlir-graphblas/issues/113.